meerk40t/core/elements/elements.py

```diff
@@ -6,6 +6,7 @@
 
 
 import contextlib
+import math
```

Comment:
pointless

Reply:
Came from a previous version, before I internalised the ellipse code